From 399ae340855d305b70c2f38e723efe56b163435a Mon Sep 17 00:00:00 2001
From: Sami Shalayel <sami.shalayel@qt.io>
Date: Tue, 7 Oct 2025 12:59:01 +0200
Subject: [PATCH 156/239] doc: fix qmltypes description: its QML, not JSON

Fix the description: qmltypes files are made out of QML, not JSON.

Pick-to: 6.8
Change-Id: I42df0adb06ffebc4e58f12db280dc496e1c69c38
Reviewed-by: Ulf Hermann <ulf.hermann@qt.io>
Reviewed-by: Fabian Kosmale <fabian.kosmale@qt.io>
Reviewed-by: Alexei Cazacov <alexei.cazacov@qt.io>
(cherry picked from commit 49b0d5d14cc53b650bbcbc79f2597a8028edd38a)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 src/qml/doc/src/tools/qtqml-tooling-qmltyperegistrar.qdoc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/qml/doc/src/tools/qtqml-tooling-qmltyperegistrar.qdoc b/src/qml/doc/src/tools/qtqml-tooling-qmltyperegistrar.qdoc
index 8db58bd87a..fc696b7488 100644
--- a/src/qml/doc/src/tools/qtqml-tooling-qmltyperegistrar.qdoc
+++ b/src/qml/doc/src/tools/qtqml-tooling-qmltyperegistrar.qdoc
@@ -9,7 +9,7 @@
 \ingroup qtqml-tooling-internal
 
 \c qmltyperegistrar generates C++ code to register C++ classes to QML and a
-\c{.qmltypes} file containing a JSON description of the classes.
+\c{.qmltypes} file containing a QML description of the classes.
 
 \c qmltyperegistrar is an internal tool shipped with Qt Qml. It is meant to be
 invoked by the build system only, you don't need to execute it directly.
-- 
2.51.2

